x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/apm_403_response_to_a_post.json

```diff
@@ -12,7 +12,7 @@
   "language": "kuery",
   "license": "Elastic License v2",
   "name": "Web Application Suspicious Activity: POST Request Declined",
-  "query": "http.response.status_code:403 and http.request.method:post",
+  "query": "http.response.status_code:403 and http.request.method:post\n",
```

Comment:
Can you leave a note here on why you had to incorporate the rule changes here.

Also, just to confirm, these are now aligned with rule version hashes from the repo?

Reply:
Matches the fleet package 0.13.1